### ISSUE TITLE
Migration to 11.0

### ISSUE THREAD

**Issue by GSLabIt** [bascouba/mymodules#1]
Hi there,

I'd like to contribute migration to v. 11.0; any chance to share ses developer documentation? 

Thx

**Comment by FlyingPigSolutions** [comment]
Is there any news regarding this migration?
We would be very much interested by it. Thanks
@GSLabIt @bascouba

**Comment by GSLabIt** [comment]
@flyingpigsolutions nope.. at least from my side.. basically need info on tags.. do u have spec?

**Comment by bascouba** [comment]
Hello guys, sorry I am not very active here and didn't see your first post. I don't work anymore at ses and try to get someone to redirect you to.

**Comment by GSLabIt** [comment]
@bascouba thanks.. if possible, need specification to work on it

**Comment by GSLabIt** [comment]
@bascouba thanks.. if possible, need specification to try work on it.

**Comment by FlyingPigSolutions** [comment]
Thanks @bascouba That would be very helpful.
Waiting for your news then.

**Comment by bascouba** [comment]
Hello again, I didn't receive any response from the company, sorry about that.
You can still try their contact platform http://www.ses-imagotag.com/contact/

**Comment by FlyingPigSolutions** [comment]
@bascouba we got contact from the sales team and they seem not aware that they have developed this internal. They are asking us to develop it on our own.
To us, the issue we face installing the version 10 of the module on odoo 11 is a label tag that is not found thus it creates an error.
@bascouba do you have a contact in the SES IMAGOTAG tech team you worked with we could contact as we are not moving on with the sales team from their side.
Without spec we can't do much
